flair/trainers/language_model_trainer.py

```diff
@@ -161,10 +135,22 @@ def tokenize(self, path: Path):
                     token += 1
 
         return ids
-
-
+        
+    
+class TextCorpus(object):
+    def __init__(self, path: Path, dictionary: Dictionary, forward: bool = True, character_level: bool = True, random_case_flip : bool = True):
+        self.dictionary: Dictionary = dictionary
+        self.forward                = forward
+        self.split_on_char          = character_level
+        self.random_case_flip       = random_case_flip
+        
+        self.train = TextDataset(path / 'train',dictionary, True,self.forward,self.split_on_char,self.random_case_flip)
```

Comment:
Should be `self.train = TextDataset(path / 'train', dictionary, False, self.forward, self.split_on_char,                                 self.random_case_flip)`, i.e. `expand_vocab` needs to be `False` to be consistent with previous behavior. Could throw errors otherwise for new characters that are not in the pre-defined dictionary.

Reply:
good catch. I will fix.

Reply:
I'm new to github's review functionality. I committed a change swapping the bool and see the change in 2c937f5. Let me know if there is a certain way you would like me to push the changes.